gatsby/plugins/gatsby-plugin-code-tabs/index.js

```diff
@@ -1,4 +1,4 @@
-const visit = require("unist-util-visit");
+const visit = require('unist-util-visit');
```

Comment:
all of this is just prettier, not sure how its diff here than in the other repo but i didnt care to figure it out